src/components/common/Carousel.jsx

```diff
@@ -0,0 +1,149 @@
+  },
+  {
+    url: "/website-v2/cohort/lbtcl.png",
+    title: "Bitcoin Core Review Club",
+    content:
+      "You get hand-on experience into the inner workings of Bitcoin Core and understand how experienced contributors review PRs. You’ll also develop a deeper understanding of Bitcoin’s design principles while sharpening your adversarial-thinking skills.",
+  },
+  {
+    url: "/website-v2/activities/cohort.png",
+    title: "Discord Community",
+    content:
+      "Come and become a part of our vibrant community of Bitcoin enthusiasts. Here, you can engage in discussions, challenge ideas, and explore Bitcoin opportunities with like-minded peers passionate about shaping the future of decentralized finance.",
+  },
+];
+
+function Carousel() {
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  const prevSlide = () => {
+    const isFirstSlide = currentIndex === 0;
+    const newIndex = isFirstSlide
+      ? slides.length - 1
+      : currentIndex - 1;
+    setCurrentIndex(newIndex);
+  };
+
+  const nextSlide = () => {
+    const isLastSlide = currentIndex === slides.length - 1;
+    const newIndex = isLastSlide ? 0 : currentIndex + 1;
+    setCurrentIndex(newIndex);
+  };
+
+  const goToSlide = (slideIndex) => {
+    setCurrentIndex(slideIndex);
+  };
+
+  return (
+    <>
+      <div className="my-2 flex flex-col lg:hidden  ">
+        {slides.map((slide) => {
+          return (
+            <div className="my-2 rounded-lg bg-orange p-3">
+              <h1 className="mb-5  font-header text-2xl  lg:p-11 lg:text-8xl">
+                {slide.title}
+              </h1>
+              <h1
+                style={{
+                  WebkitTextStrokeColor: "black",
+                  WebkitTextFillColor: "transparent",
+                  WebkitTextStrokeWidth: "2px",
+                }}
+                className="absolute top-12 mb-2 hidden font-header text-2xl opacity-10  lg:p-11 lg:text-9xl xl:flex"
+              >
+                {slide.title}
+              </h1>
+              <div className="flex flex-col lg:flex-row">
+                <div
+                  style={{
+                    backgroundImage: `url(${slide.url})`,
+                  }}
+                  className=" h-40 rounded-2xl bg-contain bg-center bg-no-repeat duration-500 lg:h-96 lg:w-1/2"
+                ></div>
+                <div className="flex  flex-col lg:w-1/2">
+                  <div className="my-5 lg:p-5 lg:text-3xl">
+                    {slide.content}
+                  </div>
+                  <button className="mt-5 rounded-lg border-2 border-dotted  border-black py-4 text-xl  hover:bg-white lg:mx-28">
+                    Tell Me More!
+                  </button>
+                </div>
+              </div>
+            </div>
+          );
+        })}
+      </div>
+
+      <div className="group relative m-auto hidden w-full rounded-2xl bg-orange px-4 py-16  lg:block">
```

Comment:
Can we split this and the earlier out into a separate components? in the same file itself? We can have two components and one parent component that hides and shows the correct carousel. 

Or maybe we can split out the card, since the card is same for both and then use that card within yhe desktop view and mobile. 

Just code refactor. This file looks too big and complicated with lots of repetition